aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md

```diff
@@ -68,17 +68,17 @@ Start Visual Studio. From the **File** menu, click **New Project**.
 
 In the **New Project** dialog box, expand **Visual C#**. Under **Installed Templates**, select **Cloud** and then select **Windows Azure Cloud Service**. Keep the default .NET Framework 4.5. Name the application ChatService and click **OK**.
 
-![Window titled 'New Project'. 'Cloud' is selected and "ChatService" is typed under 'Name'.](scaleout-with-windows-azure-service-bus/_static/image4.png)
+![Screenshot of window with Cloud selected and ChatService typed under Name.](scaleout-with-windows-azure-service-bus/_static/image4.png)
```

Comment:
How about, "Screenshot that shows the New Project dialog box. Cloud is selected in the Office Share Point folder. Chat Service is in the Name field."